metrics.py

```diff
@@ -67,41 +67,40 @@ def document_count_by(
         column: str,
         var_name: str) -> DataFrame:
     """Count documents by grouping column."""
-    # out = df.withColumn('datasourceId', lit('all'))
     out = df.groupBy(column).count().alias('count')
     out = out.withColumn('variable', f.lit(var_name))
     out = out.withColumn('field', f.lit(None).cast(t.StringType()))
     return out
 
 
-def evidence_not_null_fields_count(
-        df: DataFrame,
-        var_name: str) -> DataFrame:
-    """Count number of evidences with not null values in variable."""
-    # flatten dataframe schema
+def not_null_fields_count(df: DataFrame, var_name: str, group_by_datasource: bool) -> DataFrame:
```

Comment:
To follow the same style as the other functions.

```suggestion
def not_null_fields_count(
        df: DataFrame,
        var_name: str,
        group_by_datasource: bool) -> DataFrame:
```

Reply:
Good point. I've looked online a bit, and I found that this style is often used for signatures with type hints; almost identical, but the return value is also on a newline:

```python
def not_null_fields_count(
        df: DataFrame,
        var_name: str,
        group_by_datasource: bool
) -> DataFrame:
```

I've reformatted all functions with type hints in https://github.com/opentargets/ot-release-metrics/pull/4/commits/9c2d36e068315212078aef139e19faf4e4d2fffc; do you like it?